types/plotly.js/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for plotly.js 1.54
+// Type definitions for plotly.js 2.12
```

Comment:
there were breaking changes, in v2.0.0, for example this symbol should be removed IMO:
```ts
export const d3: typeof _d3;
```
https://github.com/plotly/plotly.js/blob/HEAD/CHANGELOG.md#removed
(unless added aback in recent versions).
the point is: I think the breaking changes should be applied, otherwise TS version will derive from native version.